### DISCUSSION TITLE
Is it an error?

### DISCUSSION THREAD

**Discussion by matheusdiogenesandrade** [cvanaret/Uno#7] [General]
I suppose that the code line [433](https://github.com/cvanaret/Uno/blob/main/uno/ingredients/subproblem/interior_point_methods/PrimalDualInteriorPointSubproblem.cpp#L433).

```cpp
assert(is_finite(this->lower_delta_z[i]) && "The displacement lower_delta_z is infinite");
```

actually should be 

```cpp
assert(is_finite(this->lower_delta_z[variable_index]) && "The displacement lower_delta_z is infinite");
```

is it right?

**Comment by cvanaret** [comment]
Indeed! This was fixed in commit 61426f5.
